Source/Core/Core/TimePlayed.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include "Common/CommonTypes.h"
+#include "Common/IniFile.h"
+
+class TimePlayed
+{
+public:
+  TimePlayed();
+  TimePlayed(std::string game_id);
+
+  template <class Rep, class Period>
+  void AddTime(std::chrono::duration<Rep, Period> time_emulated)
+  {
+    if (m_game_id == "")
+    {
+      return;
+    }
+
+    u64 previous_time;
+    m_time_list->Get(m_game_id, &previous_time);
+    m_time_list->Set(m_game_id, previous_time + u64(time_emulated.count()));
+    m_ini.Save(m_ini_path);
+  }
+
+  u64 GetTimePlayed() const;
+  u64 GetTimePlayed(std::string game_id) const;
```

Comment:
These should return the same thing that `AddTime()` takes, so probably `std::chrono::seconds` or `std::chrono::milliseconds`.